### PR TITLE
add the map of university of Tsukuba

### PR DESCRIPTION
**Pull request by sgk-000**
I added the map of university of Tsukuba.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 | [map_tc18_furo](#tc2018-furo-map-data)                                        | fuRo                          | Map    | Tsukuba Challenge 2018 Course |
 | [tsudanuma20_cit](#tsudanuma-2020-chiba-institute-of-technology-sensor-data)  | Chiba Institute of Technology | Sensor | Tsudanuma 2020                |
 | [map_tsudanuma20_cit](#tsudanuma-2020-chiba-institute-of-technology-map-data) | Chiba Institute of Technology | Map    | Tsudanuma 2020                |
-
+| [map_utsukuba22_roboken](#utsukuba-2022-roboken-map-data)                     | ROBOKEN                       | Map    | Tsukuba campus 2022           |
```
Comment (hara-y) [inline README.md]: - Add above "tsudanuma20_cit" (datasets in descending order of time).
- Change "ROBOKEN" to "University of Tsukuba".

File: README.md [hunk 2]
```diff
@@ -235,6 +235,24 @@ If you use our dataset in your academic work, please cite the following paper [[
 - **Description:** Tsudanuma Campus of Chiba Institute of Technology.
 - **License:** TBD
 
+### UTsukuba 2022, ROBOKEN, Map Data
```
Comment (hara-y) [inline README.md]: - Add above "tsudanuma20_cit" (datasets in descending order of time).
- Change "ROBOKEN" to "University of Tsukuba".

File: README.md [hunk 3]
```diff
@@ -235,6 +235,24 @@ If you use our dataset in your academic work, please cite the following paper [[
...
+
+- **Short Name:** map_utsukuba22_roboken
+- **Provider (Team):** ROBOKEN
```
Comment (hara-y) [inline README.md]: - Change "ROBOKEN" to "University of Tsukuba".

File: README.md [hunk 4]
```diff
@@ -235,6 +235,24 @@ If you use our dataset in your academic work, please cite the following paper [[
...
+- **Format:** pcd
+- **Number of Points:** 2,875,954
+- **Point Type:**
+  - **XYZ:** Yes
+  - **Intensity:** Yes
+  - **Color:** No
+  - **Normal:** Yes
+- **SLAM Method:** LIO-SAM
+- **Description:** Tsukuba Campus of University of Tsukuba.
+- **License:** TBD
```
Comment (hara-y) [inline README.md]: - Set your license. "CC BY-NC-SA 4.0" is recommended.

**Review by hara-y — CHANGES_REQUESTED** [review]
Thanks for your contribution!

I would like to request some corrections regarding the following comments.

File: README.md (hunk 1 above)
Comment (sgk-000) [reply README.md]: Added sensor data.
Fixed affiliation name.

File: README.md (hunk 2 above)
Comment (sgk-000) [reply README.md]: Added sensor data.
Fixed affiliation name.

File: README.md (hunk 3 above)
Comment (sgk-000) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (sgk-000) [reply README.md]: Added the license

**Comment by sgk-000** [comment]
I replaced the map data with a bigger one.

File: README.md [hunk 5]
```diff
@@ -13,7 +13,8 @@
 | [map_tc18_furo](#tc2018-furo-map-data)                                        | fuRo                          | Map    | Tsukuba Challenge 2018 Course |
 | [tsudanuma20_cit](#tsudanuma-2020-chiba-institute-of-technology-sensor-data)  | Chiba Institute of Technology | Sensor | Tsudanuma 2020                |
 | [map_tsudanuma20_cit](#tsudanuma-2020-chiba-institute-of-technology-map-data) | Chiba Institute of Technology | Map    | Tsudanuma 2020                |
-
+| [utsukuba22_university_of_tsukuba](#utsukuba-2022-university-of-tsukuba-sensor-data)                     | University of Tsukuba                       | Sensor    | Tsukuba campus 2022           |
```
Comment (hara-y) [inline README.md:14]: Please move the entry position of your two datasets (sensor and map) between [map_tc18_furo] and [tsudanuma20_cit].

あなたの2つのデータセット（センサと地図）の記入位置を、[map_tc18_furo] と [tsudanuma20_cit] の間に移動してください。

File: README.md [hunk 6]
```diff
@@ -235,6 +236,46 @@ If you use our dataset in your academic work, please cite the following paper [[
 - **Description:** Tsudanuma Campus of Chiba Institute of Technology.
 - **License:** TBD
 
+### UTsukuba 2022, University of Tsukuba, Sensor Data
```
Comment (hara-y) [inline README.md]: Please move the entry position of your two datasets (sensor and map) between [map_tc18_furo] and [tsudanuma20_cit].

あなたの2つのデータセット（センサと地図）の記入位置を、[map_tc18_furo] と [tsudanuma20_cit] の間に移動してください。

**Review by hara-y — CHANGES_REQUESTED** [review]
Could you please correct the insertion position of your datasets according to the following comments?

以下のコメントに従って、あなたのデータセットの挿入位置を修正して頂けますか？

**Comment by sgk-000** [comment]
> Could you please correct the insertion position of your datasets according to the following comments?
> 
> 以下のコメントに従って、あなたのデータセットの挿入位置を修正して頂けますか？

Sorry, I misunderstood your previous reply.

File: README.md (hunk 5 above)
Comment (sgk-000) [reply README.md:14]: Fixed

File: README.md (hunk 6 above)
Comment (sgk-000) [reply README.md]: Fixed

**Review by hara-y — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**